tests/utils/test_rest_utils.py

```diff
@@ -44,7 +44,7 @@ def test_non_json_ok_response():
             match="API request to endpoint was successful but the response body was not "
             "in a valid JSON format",
         ):
-            call_endpoint(host_only, "/my/endpoint", "GET", "", response_proto)
+            call_endpoint(host_only, "/api/2.0/fetch-model", "GET", "", response_proto)
```

Comment:
Fix the endpoint of this test to enable the JSON validation.

Reply:
Thanks! Can we also test that receiving a non-JSON response on a non-/api/2.0 endpoint (e.g. `/dbfs/some/path`) does not throw?